src/test/java/uk/gov/ons/ctp/integration/contactcentresvc/service/impl/CaseServiceImplModifyCaseTest.java

```diff
@@ -122,15 +136,6 @@ public void shouldAcceptCompatibleCaseTypeAndEstabType() throws Exception {
     verifyRmCaseCall(2);
   }
 
-  @Test
```

Comment:
We need some additional tests that test the following:
- existing OTHER can be modified with  estabType request of OTHER
- existing unrecognised estabType (eg "floating palace") can be modified with estabType request of OTHER
- the same as above but going through AddressType changed path rather than Address Modified logic

Reply:
Done